src/model.ts

```diff
@@ -206,237 +234,203 @@ export class ThemeEditorModel extends VDomModel {
-          minimum: 6,
-          maximum: 30
-        },
-        'content-font-size': {
-          title: 'Content font size',
-          type: 'integer',
-          minimum: 6,
-          maximum: 30
-        },
-        'code-font-size': {
-          title: 'Code font size',
-          type: 'integer',
-          minimum: 6,
-          maximum: 30
-        },
-        'border-width': {
-          title: 'Border width',
-          type: 'integer',
-          minimum: 1,
-          maximum: 10
-        },
-        'border-radius': {
-          title: 'Border radius',
-          type: 'integer',
-          minimum: 1,
-          maximum: 10
-        },
-        'ui-font-family': {
-          title: 'User Interface Font family',
-          type: 'string',
-          enum: this._fontList
-        },
-        'content-font-family': {
-          title: 'Content Font family',
-          type: 'string',
-          enum: this._fontList
-        },
-        'code-font-family': {
-          title: 'Code Font family',
-          type: 'string',
-          enum: this._codeFontList
-        }
-      }
-    };
-
-    this._formData = {
-      'ui-font-family': 'helvetica',
-      'content-font-family': 'system-ui',
-      'code-font-family': 'consolas',
-      'ui-font-size': getFontSizeFromCSSProperties(
-        CSSInfos,
-        '--jp-ui-font-size',
-        1
-      ),
-      'content-font-size': getFontSizeFromCSSProperties(
-        CSSInfos,
-        '--jp-content-font-size',
-        1
-      ),
-      'code-font-size': getNumberFromCSSProperties(
-        CSSInfos,
-        '--jp-code-font-size'
-      ),
-      'border-width': getNumberFromCSSProperties(CSSInfos, '--jp-border-width'),
-      'border-radius': getNumberFromCSSProperties(
-        CSSInfos,
-        '--jp-border-radius'
-      ),
-      'layout-color': getHexColorFromCSSProperties(
-        CSSInfos,
-        '--jp-layout-color1'
-      ),
-      'accent-color': getHexColorFromCSSProperties(
-        CSSInfos,
-        '--jp-accent-color1'
-      ),
-      'border-color': getHexColorFromCSSProperties(
-        CSSInfos,
-        '--jp-border-color1'
-      ),
-      'brand-color': getHexColorFromCSSProperties(
-        CSSInfos,
-        '--jp-brand-color1'
-      ),
-      'error-color': getHexColorFromCSSProperties(
-        CSSInfos,
-        '--jp-error-color1'
-      ),
-      'info-color': getHexColorFromCSSProperties(CSSInfos, '--jp-info-color1'),
-      'success-color': getHexColorFromCSSProperties(
-        CSSInfos,
-        '--jp-success-color1'
-      ),
-      'warn-color': getHexColorFromCSSProperties(CSSInfos, '--jp-warn-color1')
-    };
-
+    this._schema = Schema;
+    this._formData = initializeFormData();
+    this._cssProperties = {};
     this._formDataSetter = {
       'ui-font-family': (value: string) => {
-        document.body.style.setProperty('--jp-ui-font-family', value);
+        return (this._cssProperties['--jp-ui-font-family'] = value);
```

Comment:
You don't need the `return` in all those functions. There are setters, so it is usual they don't return anything.


```suggestion
        this._cssProperties['--jp-ui-font-family'] = value;
```